test/common/include/alpaka/test/mem/view/Iterator.hpp

```diff
@@ -0,0 +1,200 @@
+                            
+
+                            Elem * ptr = m_nativePtr;
+                            
+                            for(TSize dim_i = 0; dim_i < TDim::value - 1; dim_i++)
+                            {
+                                ptr += (currentIdxDimx[dim_i] * m_pitchBytes[dim_i+1]) / sizeof(Elem);
+                            }
+                            
+                            ptr += currentIdxDimx[TDim::value - 1];
+
+                            return *ptr;
+                        }
+
+                        Elem * m_nativePtr;
+                        TSize  m_currentIdx;
+                        alpaka::Vec<TDim, TSize> const m_extents;
+                        alpaka::Vec<TDim, TSize> const m_pitchBytes;
+                    
+                    };                    
+
+                    //#############################################################################
+                    //! 
+                    //#############################################################################
+                    template<
+                        typename TView,
+                        typename TSfinae = void>
+                    struct Begin
+                    {
+                        ALPAKA_FN_HOST_ACC static auto begin(TView & view)
+                            -> Iterator<TView>
+                        {
+                            return Iterator<TView>(view);
+                        }
+                    };
+                    
+
+                    //#############################################################################
+                    //! 
+                    //#############################################################################
+                    template<
+                        typename TView,
+                        typename TSfinae = void>
+                    struct End
+                    {
+
+                        ALPAKA_FN_HOST_ACC static auto end(TView & view)
+                            -> Iterator<TView>
+                        {
+                            auto extents = alpaka::extent::getExtents(view);
+                            return Iterator<TView>(view, extents.prod());
+                        }
+                    };                    
+                }
+
+                //#############################################################################
+                //! 
+                //#############################################################################
+                template<
+                    typename TView>
+                using Iterator = traits::IteratorType<TView>;
+                    
+                //#############################################################################
+                //! 
+                //#############################################################################
+                template<
+                    typename TView
+                    >
```

Comment:
Either remove one indentation level of the `>` or close it on the same line (as done in the alias template above) consistently.